cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5597,7 +5597,7 @@ search = `cs_gws_login_events` "event.name"="login_success" `cs_public_ips(ipAdd
 | `cs_current_week_login_count(gws:reports:login)` \
 | `cs_country_login_percentage` \
 | fillnull Country, Region, City value="-" \
-| eval cyences_severity = case(percentage_login_from_country==0, "high", true(), "medium") \
+| eval cyences_severity = case(percentage_login_from_country==0, "critical", true(), "medium") \
```

Comment:
This should be similar severity is Office 365 unusual login alerts.

Reply:
cyences_severity = case(percentage_login_from_country==0, "high", user_type=="Admin" or user_type=="DcAdmin", "high", true(), "medium")
Office 365 unusual login alert has **high** severity. As per the commit severity reduced from "critical" to "high" for the new location login alert.

should we keep critical or change to high?

Reply:
Its fine lets keep it critical for now. we'll sync all together later.